rust/connlib/clients/apple/src/lib.rs

```diff
@@ -129,6 +128,8 @@ impl Callbacks for CallbackHandler {
         tunnel_address_v4: Ipv4Addr,
         tunnel_address_v6: Ipv6Addr,
         dns_addresses: Vec<IpAddr>,
+        route_list_4: Vec<Ipv4Network>,
```

Comment:
(nit) the existing `tunnel_address_` params include a `v` before the IP version. Should we keep that consistent?

Reply:
I just copy pasted them from the previous callback but I can adjust them!

Reply:
Ah I see, yeah it looks like the naming of these is just inconsistent in general, so not a biggie.